media.css

```diff
@@ -0,0 +1,457 @@
+@media (min-width: 768px) and (max-width: 1023px) {
+  .wrapper {
+    width: 768px;
+    }
+
+  .header {
+    grid-template-columns: 1fr 1fr;
+    
+    /* margin-bottom: 33px;
+    padding-top: 24px; */
+    padding-bottom: 0;
+    margin-bottom: 26px;
+  }
+
+  .header__menu,
+  .header__search,
+  .header__user-block,
+  .header__cart,
+  .header__language-block {
+    display: none;
+  }
+
+  .header__logo-image {
+    display: none;
+  }
+
+  .header__logo-image_adaptive {
+    width: 156px;
+    height: 34px;
+    display: flex;
+    align-items: center;
+    margin-left: 43px;
+  }
+  
+  .header__menu-bar {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+  }
+
+  .menu-bar {
+    display: block;
+    margin-right: 44px;
+  }
+
+  .top-content {
+    background-image: url(img/header-img-768.png);
+    width: 100%;
+    height: 415px;
+    margin-bottom: 67px;
+  }
+
+ .top-content__block {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)